steps/embrace-symbol-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,38 @@
+title: Embrace iOS Symbol Upload
+summary: |
+  Upload iOS debug symbols to Embrace through Bitrise CI integration
+description: |
+  Upload iOS debug symbols to Embrace through Bitrise CI. This will make your integration with Embrace more efficient. Environment Variables are supported via the upload tool.
+
+  Env Vars:
+  ---------
+  EMBRACE_APP: Your Embrace app idenifier
+  EMBRACE_API_TOKEN: Your Embrace upload API token
+website: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload
+source_code_url: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload
+support_url: https://github.com/embrace-io/bitrise-step-embrace-symbol-upload/issues
+published_at: 2023-10-05T14:49:30.418918-04:00
+source:
+  git: https://github.com/embrace-io/bitrise-symbol-upload.git
+  commit: f24fc1c9e96311333c2deab2ae69f3ad4b13409b
+project_type_tags:
+- ios
```

Comment:
I think this step is relevant for cross-platform technologies too (when the project is targeting iOS), we recommend listing the relevant cross-platform types from here: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#platforms

Reply:
Got it, thank you!